osbrain/tests/test_agent.py

```diff
@@ -599,16 +603,18 @@ def test_running_exception(nsproxy):
     assert not agent.is_running()
 
 
+@skip_windows_addr
 def test_agent_error_address_already_in_use(nsproxy):
     """
     Running an agent should raise an error if address is already in use.
     """
     with pytest.raises(RuntimeError) as error:
         run_agent('a0', nsaddr=nsproxy.addr(), addr=nsproxy.addr())
     assert 'OSError' in str(error.value)
-    assert 'Address already in use' in str(error.value)
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason='Windows allows binding to any port')
```

Comment:
Even if only used once, I'd declare all Windows skips the same way (with a `@skip...` decorator). Will make finding all Windows skips easier.